### DISCUSSION TITLE
Replace Tweet-prompt with Fediverse/other-prompt

### DISCUSSION THREAD

**Discussion by lucahammer** [community/community#70016] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After sponsoring a project, there is a box that prompts users to tweet about becoming a sponsor, but since Twitter was bought by X Corp. it's losing relevancy. I would prefer the prompt to be used for a more relevant platform or even making it dynamic based on profile URLs: If someone has a fediverse link in their profile, use that to generate a post-prompt that works with exactly that instance, if they have a Bluesky link, show that one. 

![image](https://github.com/community/community/assets/1390793/89b867fc-e3e9-469d-8e10-30fa71cbf929)

**Comment by anjuan** [comment]
Hi, @lucahammer, I'm the Engineering Manager for GitHub Sponsors. I can see how providing other ways to show you've become a sponsor would be helpful. I like your idea of leveraging the profile links to dynamically provide options. This isn't something on our immediate roadmap, but I'll keep it in mind for possible future work by the team.

**Comment by driehuis** [comment]
I hate to write an &lt;aol>Me Too!&lt;/aol>, but I was rather taken aback when after sponsoring a project I got invited to share my good vibes at the dead bird site. You may want to revisit your decision to have only one option for sharing ones enthusiasm for sponsoring.
